core/config/config.go

```diff
@@ -86,8 +92,8 @@ func Load(ctx context.Context, db pg.DB, rDB *raft.Service) (*Config, error) {
 				// If we got this far but failed to delete from PG, it's really NBD. Just
 				// log the failure and carry on.
 				log.Error(ctx, err, "failed to delete config from postgres")
-
 			}
+			err = migrateAccessTokens(ctx, db, rDB)
```

Comment:
Oh, yes, this was a question I wanted to address. If we fail to migrate the tokens, it means that they can't be used on this core. So what should we do here?